packages/app/src/__deprecated__/renderer/locales/default/en-US.json

```diff
@@ -383,25 +383,12 @@
   "module.messages.contactTooltipDescription": "Contact details",
   "module.messages.conversationDelete": "1 conversation was deleted.",
   "module.messages.conversations": "Conversations",
-  "module.messages.conversationsDelete": "{number} conversations were deleted.",
   "module.messages.conversationsSelectionsNumber": "{num, plural, =-1 {All Conversations} one {# Conversation} other {# Conversations}} selected",
   "module.messages.deleteButton": "Delete",
-  "module.messages.deleteModalBody": "Do you really want to delete <b>{num, plural, =-1 {all conversations} one {# conversation} other {# conversations}}?</b>",
```

Comment:
Not sure if we should delete translations. They can still be in use for users with lower center versions.

Reply:
Yes, it can be. But the lower MC's version has their of set of translations